test/e2e/apibinding/apibinding_test.go

```diff
@@ -248,13 +220,45 @@ func TestAPIBinding(t *testing.T) {
 		}, wait.ForeverTestTimeout, 100*time.Millisecond, "expected naming conflict")
 	}
 
+	verifyVirtualWorkspaceURLs := func(serviceProviderWorkspace logicalcluster.Name) {
```

Comment:
Is this really the only test that needs it? I would have expected every test that previously waited on the export to be bound to now wait on this?

Reply:
It's the only test that fails; I was honestly surprised too. It appears to be the only one that waits for the VirtualWorkspaceURL explicitly.

Reply:
I guess I meant the rest need to wait explicitly - do they handle `""` correctly? Are there no other ones that use a virtual workspace?

Reply:
Looking at them again, we actually worked around it by making APIBindings ~immediately after the APIExports.

I added an explicit check in the virtual workspaces e2e tests to make sure the field is empty before APIBindings are created.